oarepo_ui/theme/assets/semantic-ui/less/oarepo_ui/themes/default/elements/label.overrides

```diff
@@ -0,0 +1,17 @@
+.ui.label {
+  &.active-filter-label {
+    font-weight: normal;
+    color: black;
+  }
+  padding: 0.5rem 1rem;
```

Comment:
This has potential to break sites ui, do  we really need to do this here?

Reply:
@mirekys My bad, this was supposed to be inside of the active-filter-label selector. Do you generally disagree with keeping the styles here or just with its placement directly below ui.label? My logic was that, as component is in oarepo ui, it would be ideal if it did not require someone to place styles in the app itself, in order for it to look reasonable. Do you have something specific in mind please or is it enough to place it inside the active-filter-label selector?

Reply:
I'd like to have it under className="ui active filter label" classes, the point here was just to avoid changing appearance of completely unrelated labels here (all "ui label" elements for everyone using oarepo-ui), if not really necessary